lib/axe-injector.js

```diff
@@ -1,12 +1,16 @@
 const { StaleElementReferenceError } = require('selenium-webdriver').error;
 
 class AxeInjector {
-  constructor({ driver, config, axeSource = null, options = {} }) {
```

Comment:
I didn't want to pass it thru options since it looks like options was an axe-core specific flag. 
```
/**
 * Options to directly pass to `axe.run`.  See API documentation for axe-core for use.  Will override any other configured options, including calls to `withRules` and `withTags`.
 * @param  {Object} options Options object
 * @return {AxeBuilder}
 */
AxeBuilder.prototype.options = function(options) {
  this._options = options;
  return this;
};```